### PR TITLE
feat(github): add feature request template

### PR DESCRIPTION
**Pull request by laurent-laporte-pro**

### CODE REVIEW

**Comment by hdinia** [comment]
## Description

Provide a clear and concise description of the requested feature.

## Problem to Solve

Explain the problem that this feature is trying to solve or the use case that it is trying to support.

## Proposed Solution

Describe how you envision this feature being implemented.

## Additional Context

Add any other context or examples to help explain the requested feature, including:
- Screenshots that illustrate the issue or desired outcome
- Existing parts of the feature in Antares Simulator, if applicable

## Possible Alternatives

If you have considered alternative solutions to the problem, describe them here.

## Impact and Priority
- How important is this feature for your workflows or use case?
- Impact:
  - [ ] Low
  - [ ] Medium
  - [ ] High
  
- How would the lack of this feature impact your ability to achieve your goals?
- Priority:
  - [ ] Low
  - [ ] Medium
  - [ ] High

## Developer Notes

For the developers working on this feature request:

- Estimated time to implement (in days): 
- Difficulty:
  - [ ] Easy
  - [ ] Moderate
  - [ ] Hard
- Any potential roadblocks or concerns: [e.g. dependencies on other features, limitations in technology]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (skamril) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Add default label

**Review by skamril — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+assignees: ''
+
+---
+
```
Comment (skamril) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: "Description", "Problem to Solve" and "Proposed Solution" cannot be merged in one?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Add any other context or examples to help explain the requested feature, including:
+- Screenshots that illustrate the issue or desired outcome
+- Existing parts of the feature in Antares Simulator, if applicable
+
+## Possible Alternatives
+
+If you have considered alternative solutions to the problem, describe them here.
+
+## Impact and Priority
```
Comment (skamril) [inline .github/ISSUE_TEMPLATE/feature_request.md]: For me "Impact and Priority" duplicates with labels and "Developer Notes" is useless

**Review by skamril — CHANGES_REQUESTED** [review]

**Comment by hdinia** [comment]
### Updated

## Problem Statement

A concise statement that clearly defines the problem the feature is trying to solve, or user problem it addresses:

## Proposed Solution

A high-level description of the proposed solution, including:

- The desired outcome
  
- Screenshots or mockups that illustrate the desired outcome (if available)
  
- Any specific requirements or constraints for the feature

- Add any other context or examples to help explain the requested feature

## Developer Notes

- Any potential roadblocks or concerns: [e.g. dependencies on other features, limitations in technology]

**Comment by laurent-laporte-pro** [comment]
J'aimerais plutôt que l'on présente un template qui se rapprocherait plus d'un User Story.

Exemple:

## Problem Statement
As a **[user]**, I want **[feature]**, so that **[user goal/value]**.

## Description:

### Context:
A brief description of the context in which the user wants to use this feature

### Desired behavior
A high-level description of the proposed solution, including:

- The desired outcome
- Screenshots or mockups that illustrate the desired outcome (if available)
- Any specific requirements or constraints for the feature
- Add any other context or examples to help explain the requested feature

### Acceptance criteria
The criteria that must be met for this feature to be considered complete

- The **[user]** can **[do something]** when **[a condition is filled]**
- ...

**Pull request merged**